monolith/interactors/ScryfallCard.ts

```diff
@@ -0,0 +1,123 @@
+export default interface ScryfallCard {
```

Comment:
We need to extend this with the `QOH` type as well that has inventory data attached